### PR TITLE
Add a crouton group that can access lockfiles

### LINKED ISSUE #908: trusty X server display malfunctions when uid != 1000
I have been using Crouton with `trusty` and Openbox (via the `lxde` target) on an Asus Chromebox for a little over a month.  Yesterday I rebooted, which caused Chrome OS to upgrade itself.  Following upgrade, inside `trusty` the X server display started malfunctioning.

Specifically:  Initially, most of the screen is black, the exception being the LXPanel at the bottom.  The screen only refreshes due to user activity, and even then, only partially refreshes.  Consequently, the screen is usually displaying stale and/or corrupted windows.

Interestingly, the problem only manifests when running as a user other than `1000`.

Steps to reproduce the problem:

    # Create a trusty chroot named vanilla.
    $ sudo sh -e ~/Downloads/crouton -t lxde -n vanilla -r trusty
    $ sudo startlxde -n vanilla
    # X display functions properly.

    # Then create another user named u1100.
    $ sudo enter-chroot -n vanilla
    $ sudo groupadd -g 1100 g1100
    $ sudo useradd -g g1100 -m -s /bin/bash -u 1100 u1100
    $ exit

    # Enter LXDE as user u1100.
    $ sudo startlxde -n vanilla -u u1100
    # The X display malfunctions, as described above.

Ancillary issues:

1) In LXDE, the Logout option does nothing.  In `precise` it works, but not in `trusty`.  Logging out requires switching to Chrome OS, and pressing ^C in the terminal to kill X.  (Aside: this does not really affect me, as I typically use a custom `~/.xsession` file.)

2) During the creation of the `vanilla` chroot, my keyboard layout in Chrome OS was reset from Dvorak to Qwerty.  The only way to correct was to log out of Chrome OS.

3) Simultaneous with the keyboard layout change, the responsive of my trackball dropped to almost nothing.  As in: spin and spin and spin and spin to slowly, slowly move the mouse cursor across the screen.  At this point, the Chrome OS settings page said that no mouse or touchpad was detected.  Again, logging out of Chrome OS was required to fix the problem.

Chrome OS version info:
Version 35.0.1916.155
Platform 5712.88.0 (Official Build) stable-channel panther
Firmware Google_Panther.4920.24.22

crouton version info:
VERSION='1-20140421214646~master:b766b319'

An obvious workaround should be to only run Crouton as user 1000.

### PR DESCRIPTION
**Pull request by dnschneid**
Fixes #908 if you add the user to the right groups.

### CODE REVIEW

File: chroot-bin/croutontriggerd [hunk 1]
```diff
@@ -39,6 +39,8 @@ EVENT_DEV_POLL=15
 # Only one at a time. xbindkeys lockfile is for legacy compatibility
 mkdir -m 775 -p "$CROUTONLOCKDIR"
 exec 3>>"$CROUTONLOCKDIR/xbindkeys"
+chmod -Rf g+rwX '/tmp/crouton-lock' || true
```
Comment (smibarber) [inline chroot-bin/croutontriggerd]: `"$CROUTONLOCKDIR"`?

Comment (dnschneid) [inline chroot-bin/croutontriggerd]: "$CROUTONLOCKDIR".

File: chroot-bin/croutonclip [hunk 2]
```diff
@@ -118,6 +118,8 @@ current='cros'
 
 mkdir -m 775 -p "$CROUTONLOCKDIR"
 exec 3>>"$CROUTONLOCKDIR/clip"
+chmod -Rf g+rwX "$CROUTONLOCKDIR" || true
```
Comment (drinkcat) [inline chroot-bin/croutonclip]: Huh, I did not know about `X`!

Comment (drinkcat) [inline chroot-bin/croutonclip]: (And it's POSIX!)

File: host-bin/enter-chroot [hunk 3]
```diff
@@ -575,8 +575,9 @@ fi
 # so that users have access to shared hardware, such as video and audio.
 gfile="$CHROOT/etc/group"
 if [ -f "$gfile" ]; then
-    for group in audio:hwaudio cras:audio cdrom devbroker-access disk floppy \
-                 i2c input lp serial tape tty usb:plugdev uucp video wayland; do
+    for group in audio:hwaudio cras:audio cdrom chronos-access:crouton \
```
Comment (drinkcat) [inline host-bin/enter-chroot]: Wait, why chronos-access?

File: chroot-bin/croutonclip (hunk 2 above)
Comment (dnschneid) [inline chroot-bin/croutonclip]: Yeah, that's my favorite chmod parameter.  It makes -R behave sanely.

**Comment by drinkcat** [comment]
+1

**Pull request merged**